### PR TITLE
Add thread until

### PR DESCRIPTION
**Pull request by lanza**
Didn't really read the code base too thoroughly, just kinda hacked until it works. Is there anything I'm missing?

### CODE REVIEW

File: autoload/nvimgdb.vim [hunk 1]
```diff
@@ -283,6 +287,7 @@ function! s:DefineCommands()
   command! GdbBreakpointToggle call nvimgdb#ToggleBreak()
   command! GdbBreakpointClearAll call nvimgdb#ClearBreak()
   command! GdbRun call nvimgdb#Send("run")
+  command! GdbUntil call nvimgdb#Send("thread until " . line('.'))
```
Comment (sakhnik) [inline autoload/nvimgdb.vim]: That will work in LLDB, but the command could be easily supported for the whole suite LLDB+GDB+PDB. For that, the backend-specific commands could be defined in the backend dictionary at the beginning of the script, and merely referred here.

File: autoload/nvimgdb.vim [hunk 2]
```diff
@@ -226,6 +226,7 @@ endfunction
 
 function! s:DefKeymapVars()
   " Set global key maps
+  call s:DefKeymapVar("t:nvimgdb_key_until",      "g:nvimgdb_key_until",      "<c-u>" )
```
Comment (sakhnik) [inline autoload/nvimgdb.vim]: I'd appreciate if the default keymap were something like `<f4>`, because `<c-u>` is page up in the default Vim.

Comment (lanza) [reply autoload/nvimgdb.vim]: Is this what you suggest? I've never used f keys for anything and have the idiotic TB on my MacBook and don't know which fkeys are free.

Comment (sakhnik) [reply autoload/nvimgdb.vim]: Yes, precisely. Anyone can still define whatever is most suitable for them in a personal vimrc. But by default, it'd better not overlap with Vim defaults.

**Comment by sakhnik** [comment]
@lanza, are you still working on the review comments? Or is it necessary that I finish the work?
I realize that 20% of work might be taking 80% of the time.

**Comment by lanza** [comment]
Nah, just lazy. vimscript is still mostly voodoo to my eyes.

**Comment by sakhnik** [comment]
I see you've made a good progress. The code is good. So I've just added a couple of tests and merged the branch.

**Pull request merged**